tools/benchmark/cmd/put.go

```diff
@@ -173,12 +173,12 @@ func hashKV(cmd *cobra.Command, clients []*v3.Client) {
 	rh, eh := clients[0].HashKV(context.Background(), host, 0)
```

Comment:
Another option would be to re-use `err` here and in line 178. And keep the panics as they were before.